public/app/plugins/datasource/loki/LanguageProvider.ts

```diff
@@ -127,210 +67,6 @@ export default class LokiLanguageProvider extends LanguageProvider {
     return this.labelKeys;
   }
 
-  /**
```

Comment:
This logic was fully replaced in https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/CompletionDataProvider.ts#L32